pkg/cmd/kind/load/docker-image/docker-image.go

```diff
@@ -48,13 +49,13 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) < 1 {
-				return errors.New("name of image is required")
+				return errors.New("a single image name or a space-separated list of image names is required")
 			}
 			return nil
 		},
-		Use:   "docker-image <IMAGE>",
-		Short: "Loads docker image from host into nodes",
-		Long:  "Loads docker image from host into all or specified nodes by name",
+		Use:   "docker-image <IMAGE1 IMAGE2 IMAGE3>",
```

Comment:
nit: this could be more standard by doing this instead:


```suggestion
		Use:   "docker-image <IMAGE> [IMAGE...],
```

Reply:
Not sure of the standards on this project but `<ARG>` generally denotes requires and `[ARG]` being optional, with `...` being multiple. We could probably shorten this to `<IMAGE...>` but I haven't seen that before and am not sure 😆

Reply:
aah cool cool. Got  it.